packages/rancher-cis-benchmark/charts/templates/scanprofile-rke-1.5-permissive.yml

```diff
@@ -2,7 +2,7 @@
 apiVersion: cis.cattle.io/v1
 kind: ClusterScanProfile
 metadata:
-  name: rke-profile-permissive
+  name: rke-profile-permissive-1.5
```

Comment:
question: when we upgrade from the older version of chart to this, will the ClusterScanProfile CRs named `rke-profile-permissive` and `rke-profile-hardened` get deleted by the upgrade?

Reply:
yes they got deleted by the upgrade